### PR TITLE
[NYC-2023] Initial PR to Setup Event

### PR DESCRIPTION
**Pull request by kaerus01**
This is the initial PR for NYC 2023 devopsdays! We have a couple organization membership changes.

### CODE REVIEW

File: data/events/2023-new-york-city.yml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+registration_date_start: 2023-04-01T00:00:00-05:00
+registration_date_end: 2023-06-04T00:00:00-05:00
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
```
Comment (kuritz) [inline data/events/2023-new-york-city.yml:33]:
```suggestion
coordinates: "40.771256333924214,-73.98969432566848" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
```

**Review by kuritz — COMMENTED** [review]
Couple more small things :)

File: content/events/2023-new-york-city/welcome.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      </div>
+    </div>
+    <div class = "row">
+      <div class = "col-md-1 mx-5 px-0">
+        <h4 class="my-3"><strong>Register</strong></h4>
+      </div>
+      <div class = "col-md-9">
+        <h4 class="my-3"><small>{{< event_link page="registration" text="Tickets are on sale now!" >}}</small></h4>
+      </div>
+    </div>
```
Comment (kuritz) [inline content/events/2023-new-york-city/welcome.md:52]:
```suggestion
    <!-- <div class = "row">
      <div class = "col-md-1 mx-5 px-0">
        <h4 class="my-3"><strong>Register</strong></h4>
      </div>
      <div class = "col-md-9">
        <h4 class="my-3"><small>{{< event_link page="registration" text="Tickets are on sale now!" >}}</small></h4>
      </div>
    </div> -->
```

File: data/events/2023-new-york-city.yml [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+location: "John Jay College" # Defaults to city, but you can make it the venue name.
+#
+location_address: "524 West 59th Street, New York, NY 10019" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  - name: propose
+  - name: location
+  - name: registration
```
Comment (kuritz) [inline data/events/2023-new-york-city.yml:41]:
```suggestion
  # - name: registration
```

File: content/events/2023-new-york-city/welcome.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    <div class = "row">
+      <div class = "col-md-1 mx-5 px-0">
+        <h4 class="my-3"><strong>Program</strong></h4>
+      </div>
+      <div class = "col-md-9">
+        <h4 class="my-3"><small>View the {{< event_link page="program" text="Event Schedule." >}}</small></h4>
+      </div>
+    </div>
```
Comment (kuritz) [inline content/events/2023-new-york-city/welcome.md:28]:
```suggestion
    <!-- <div class = "row">
      <div class = "col-md-1 mx-5 px-0">
        <h4 class="my-3"><strong>Program</strong></h4>
      </div>
      <div class = "col-md-9">
        <h4 class="my-3"><small>View the {{< event_link page="program" text="Event Schedule." >}}</small></h4>
      </div>
    </div> -->
```

File: content/events/2023-new-york-city/welcome.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    <!-- <div class = "row">
+      <div class = "col-md-1 mx-5 px-0">
+        <h4 class="my-3"><strong>Slack</strong></h4>
+      </div>
+      <div class = "col-md-9">
+        <h4 class="my-3"><small> Join the <a href="https://launchpass.com/nycdevops" target="_blank">NYC Devops Slack Workspace</a></small></h4>
+      </div> -->
+    </div>
```
Comment (kuritz) [inline content/events/2023-new-york-city/welcome.md:84]: think this broke the layout
```suggestion
      </div>
    </div> -->
```

File: content/events/2023-new-york-city/welcome.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+      </div>
+    </div> -->
+    <!-- <div class = "row">
+      <div class = "col-md-1 mx-5 px-0">
+        <h4 class="my-3"><strong>Propose</strong></h4>
+      </div>
+      <div class = "col-md-9">
+        <h4 class="my-3"><small>Our CFP is now closed. Sorry, we can't accept late submissions.</small></h4>
+      </div>
+    </div> -->
```
Comment (kuritz) [inline content/events/2023-new-york-city/welcome.md:60]:
```suggestion
    <!-- <div class = "row">
      <div class = "col-md-1 mx-5 px-0">
        <h4 class="my-3"><strong>Propose</strong></h4>
      </div>
      <div class = "col-md-9">
        <h4 class="my-3"><small>{{< event_link page="propose" text="Propose a talk!" >}}</small></h4>
      </div>
    </div> -->
```

File: data/events/2023-new-york-city.yml (hunk 1 above)
Comment (mattstratton) [reply data/events/2023-new-york-city.yml]: FWIW the map feature which used this field isn't in use anymore, but still good to have just in case :)

Comment (kuritz) [reply data/events/2023-new-york-city.yml]: got it, thanks! do you recommend we remove the component from the `location` page? it seems to be rendering on the preview.

https://github.com/devopsdays/devopsdays-web/pull/12466/files#diff-d9ef0e2209f4e6d81fae233ae780f45ff846de8dc1c9dcb6809e61d6daea374dR11

**Pull request closed without merging**